src/symbols.js

```diff
@@ -652,3 +652,13 @@ for (i = 0x0410; i <= 0x044F; i++) {
     ch = String.fromCharCode(i);
     defineSymbol(text, main, textord, ch, ch);
 }
+
+// Unicode versions of existing characters
+defineSymbol(text, main, textord, "\u2013", "–");
+defineSymbol(text, main, textord, "\u2014", "—");
+defineSymbol(text, main, textord, "\u2018", "‘");
+defineSymbol(text, main, textord, "\u2019", "’");
+defineSymbol(text, main, textord, "\u201c", "“");
+defineSymbol(text, main, textord, "\u201d", "”");
+defineSymbol(text, main, textord, "\u00a1", "¡");
+defineSymbol(text, main, textord, "\u00bf", "¿");
```

Comment:
The fonts don't include these, but we could render them and use a CSS transform to flip them.  You should also add metrics for these in fontMetrics.js.  I think the height/depth for `y` should be pretty close.